tests/api_test/api_test.cpp

```diff
@@ -1116,7 +1120,7 @@ TEST_CASE("load_native_program_invalid3", "[native][negative]")
 }
 TEST_CASE("load_native_program_invalid4", "[native][negative]")
```

Comment:
nit: maybe change the test case name as it is not an "invalid" program / test case.

Reply:
Done